examples/with-docker/Dockerfile

```
@@ -7,7 +7,7 @@ RUN apk add --no-cache libc6-compat
 WORKDIR /app
 
 # Install dependencies based on the preferred package manager
-COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml* ./
+COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml* .npmrc ./
```

```suggestion
COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml* .npmrc* ./
```
Since this file might not exist it needs to be a wildcard to avoid failing if it doesn't